webpack/services/mock.js

```diff
@@ -0,0 +1,218 @@
+// eslint-disable-next-line import/prefer-default-export
+export const translate = a => a;
+
+const API_REQUESTS = [
```

Comment:
I know that you were just moving this but how do you feel about converting it to match the API specification provided here:  https://gist.github.com/jlsherrill/6678dc74ae2a04fa9055c69cf3c40802?

Reply:
I set it up to match but add some more meta-data at the top level (possibly to enable mocking the api). The response key should match what's in that gist.